skills_DS/base/models.py

```diff
@@ -54,7 +54,7 @@ class Profile(models.Model):
     user = models.OneToOneField(User,on_delete=models.CASCADE, related_name="profile")
     age = models.IntegerField(null=True)
     gender = models.CharField(max_length=20)
```

Comment:
For gender, we should use enumeration types. Django's document provides a very good example on how you can achieve it: https://docs.djangoproject.com/en/4.0/ref/models/fields/#enumeration-types